lm/srilm.py

```diff
@@ -100,13 +100,16 @@ class DiscountNgramsJob(Job):
     Create a file with the discounted ngrams with SRILM
     """
 
+    __sis_hash_exclude__ = {"data_for_optimization": None}
+
     def __init__(
         self,
         ngram_order: int,
         counts: tk.Path,
         count_exe: tk.Path,
         *,
         vocab: Optional[tk.Path] = None,
+        data_for_opt: Optional[tk.Path] = None,
```

Comment:
In the optimal case the key of `__sis_hash_exclude__` would match the newly added parameter.

I would prefer "optimization" over "opt"

Reply:
yes good point ^^